installing/installing_aws/installing-aws-localzone.adoc

```diff
@@ -0,0 +1,220 @@
+:_content-type: ASSEMBLY
+[id="installing-aws-localzone"]
+= Installing a cluster on user-provisioned infrastructure in AWS that uses local zones
+include::_attributes/common-attributes.adoc[]
+:context: installing-aws-localzone
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Amazon Web Services (AWS) that uses both infrastructure that you provide and local zones.
+
+One way to create this infrastructure is to use the provided CloudFormation templates. You can modify the templates to customize your infrastructure or use the information that they contain to create AWS objects according to your company's policies.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the cloud provider and the installation process of {product-title}. Several CloudFormation templates are provided to assist in completing these steps or to help model your own. You are also free to create the required resources through other methods; the templates are just an example.
+====
+
+== Prerequisites
+
+* You reviewed details about the xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update] processes.
+* You read the documentation on xref:../../installing/installing-preparing.adoc#installing-preparing[selecting a cluster installation method and preparing it for users].
+* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster and configured local zones for your account.
```

Comment:
I believe we can add a directly link to the Opt In section, if you decide to keep the opt in on the "Aws account" page:
```suggestion
* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster
* You xref:../../installing/installing_aws/installing-aws-account.adoc#installation-aws-add-local-zone-locations_installing-aws-account[opted in the AWS Local Zone group] which the subnets will be created.
```

Reply:
Done. The exact addition I provided is: 

* You xref:../../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[configured an AWS account] to host the cluster. 
* You xref:../../installing/installing_aws/installing-aws-account.adoc#installion-aws-add-local-zone-locations_installing-aws-account[opted in to the AWS Local Zone group] in which the subnets will be created.